src/index.js

```diff
@@ -107,29 +109,37 @@ class Popover extends Component {
     ) {
       if (browserSelection.isCollapsed === false) {
         onTextSelect && onTextSelect();
-        this.setState({ isTextSelected: true });
+        this.setState({ isTextSelected: true, isOpen: true });
       } else {
         onTextUnselect && onTextUnselect();
-        this.setState({ isTextSelected: false });
+        this.setState({ isTextSelected: false, isOpen: false });
       }
 
       this.setState({ position });
     } else if (this.state.isTextSelected) {
       onTextUnselect && onTextUnselect();
-      this.setState({ isTextSelected: false });
+      this.setState({ isTextSelected: false, isOpen: true });
     }
   };
-
+  renderPopOver = style => {
+    const { children, isOpen, className, measureRef } = this.props;
+    const { position } = this.state;
+    let renderElement = null;
+    if (position !== null) {
+      let renderCondition =
+        typeof isOpen === "undefined" ? this.state.isOpen : isOpen;
+      renderElement = renderCondition ? (
+        <div key="popup" className={className} style={style} ref={measureRef}>
+          {children}
+        </div>
+      ) : (
+        renderElement
+      );
+    }
+    return renderElement;
+  };
```

Comment:
I'm a little bit sceptical of this change - I don't think one needs to change this much to use add `this.state.isOpen` to the mix, will explain below